errors/abci_test.go

```diff
@@ -64,93 +49,37 @@ func (s *abciTestSuite) TestABCInfo() {
 			wantCode:  1,
 			wantSpace: UndefinedCodespace,
 		},
-		// This is hard to test because of attached stacktrace. This
-		// case is tested in an another test.
-		// "wrapped stdlib is a full message in debug mode": {
-		//	err:      Wrap(io.EOF, "cannot read file"),
-		//	debug:    true,
-		//	wantLog:  "cannot read file: EOF",
-		//	wantCode: 1,
-		// },
-		"custom error": {
-			err:       customErr{},
-			debug:     false,
-			wantLog:   "custom",
-			wantCode:  999,
-			wantSpace: "extern",
-		},
-		"custom error in debug mode": {
-			err:       customErr{},
-			debug:     true,
-			wantLog:   "custom",
-			wantCode:  999,
-			wantSpace: "extern",
-		},
 	}
 
 	for testName, tc := range cases {
-		s.T().Run(testName, func(t *testing.T) {
+		t.Run(testName, func(t *testing.T) {
 			space, code, log := ABCIInfo(tc.err, tc.debug)
-			s.Require().Equal(tc.wantSpace, space, testName)
-			s.Require().Equal(tc.wantCode, code, testName)
-			s.Require().Equal(tc.wantLog, log, testName)
-		})
-	}
-}
-
-func (s *abciTestSuite) TestABCIInfoStacktrace() {
-	cases := map[string]struct {
-		err            error
-		debug          bool
-		wantStacktrace bool
-		wantErrMsg     string
-	}{
-		"wrapped SDK error in debug mode provides stacktrace": {
-			err:            Wrap(ErrUnauthorized, "wrapped"),
-			debug:          true,
-			wantStacktrace: true,
-			wantErrMsg:     "wrapped: unauthorized",
-		},
-		"wrapped SDK error in non-debug mode does not have stacktrace": {
-			err:            Wrap(ErrUnauthorized, "wrapped"),
-			debug:          false,
-			wantStacktrace: false,
-			wantErrMsg:     "wrapped: unauthorized",
-		},
-		"wrapped stdlib error in debug mode provides stacktrace": {
-			err:            Wrap(fmt.Errorf("stdlib"), "wrapped"),
-			debug:          true,
-			wantStacktrace: true,
-			wantErrMsg:     "wrapped: stdlib",
-		},
-	}
-
-	const thisTestSrc = "cosmossdk.io/errors.(*abciTestSuite).TestABCIInfoStacktrace"
-
-	for testName, tc := range cases {
-		s.T().Run(testName, func(t *testing.T) {
-			_, _, log := ABCIInfo(tc.err, tc.debug)
-			if !tc.wantStacktrace {
-				s.Require().Equal(tc.wantErrMsg, log, testName)
-			} else {
-				s.Require().True(strings.Contains(log, thisTestSrc), testName)
-				s.Require().True(strings.Contains(log, tc.wantErrMsg), testName)
+			if space != tc.wantSpace {
```

Comment:
personal preference: I like `"github.com/stretchr/testify/assert"` for these checks but vanilla Go testing is fine, too if you want to avoid any deps.